stock_picking_report_xlsx/__openerp__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock picking xlsx report',
+    'version': '8.0.1.0.0',
+    'summary': "Print stock picking report in xlsx",
```

Comment:
for consintency it should be better use single quote everywhere, so please put:
`'Print stock picking report in xlsx'`